server/db/helper.js

```diff
@@ -50,14 +50,41 @@ const getCourses = () => Course.findAll({
 /**
  * Gets the user's profile info.
  * @param {integer} userId - The user's ID.
+ * @param {string} userEmail- The users's Email.
+ * @param {string} googleId- The users's google id
  */
-const getUser = (userId) => User.findOne({
+const getUser = (googleId) => User.findOne({
   where: {
-    id: userId,
+    googleId,
   },
-  attributes: { exclude: ['createdAt', 'updatedAt'] },
+  attributes: { exclude: ['createdAt', 'updatedAt', 'email', 'id'] },
 });
 
+/**
+ * Gets the user's profile info.
```

Comment:
This helper seems like it creates instead of gets

Reply:
lol I copied the other helper and didnt fix the description

Reply:
thats the get user function to get user by the google id